adapters/sqlstore/sqlstore.go

```diff
@@ -135,5 +137,13 @@ func (s *SQLStore) List(ctx context.Context, workflowName string, offsetID int64
 		filterStr += fmt.Sprintf(" and run_state=%v ", filter.ByRunState().Value)
```

Comment:
how com this isn't using a parameterized value, i.e. ? instead of %v

Reply:
Good catch. Will update.